presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloMetadata.java

```diff
@@ -172,32 +172,32 @@ public void dropView(ConnectorSession session, SchemaTableName viewName)
     }
 
     @Override
-    public List<SchemaTableName> listViews(ConnectorSession session, String schemaNameOrNull)
+    public List<SchemaTableName> listViews(ConnectorSession session, Optional<String> schemaName)
```

Comment:
> `Avoid deprecated API in AccumuloMetadata`

I think this should be reworded.

`Implement Optional-based connector metadata API in Accumulo` ?